src/features/snack.ts

```diff
@@ -0,0 +1,35 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+export const SNACK = "SNACK";
+export type SnackAction = {
+  type: typeof SNACK;
+  snack: boolean;
+  severity: "success" | "error";
```

Comment:
I'm not sure if `severity` matches what those values represent but I might be wrong here.